src/us/kbase/common/utils/CondorUtils.java

```diff
@@ -100,8 +110,46 @@ public static CondorResponse runProcess(String[] condorCommand) throws IOExcepti
         return new CondorResponse(stdOutMessage, stdErrMessage);
     }
 
+
+    /**
+     * Parse out requirements frm client groups
```

Comment:
Can you add some comments about the syntax you are looking for in the input string? Looks like it is comma separated, with the first item being the clientgroup, but then you are parsing for equals and handling special cases. It would be good to have the code describe the expected syntax.

Reply:
Done.